README.md

```diff
@@ -1,70 +1,230 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<div align="center">
 
-## Available Scripts
+  <h3><b> Math Magician</b></h3>
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/228166320-8e83d84d-bf21-4a14-bfad-c31551f65d7c.png)

- [ ] Kindly remove the highlighted title because you already have a title after Table of Contents 🤔 It would be very great if you remove it and make your readme file more professional and descriptive 😊